lib/govuk_content_schemas/allowed_document_types.yml

```diff
@@ -35,16 +33,12 @@
 - esi_fund
 - fatality_notice
 - field_of_operation
-- financial_release
-- financial_releases_campaign
 - financial_releases_geoblocker
```

Comment:
It's a shame this and the `financial_releases_success` document type aren't gone from the content store, they don't appear to do geoblocking any more https://www.gov.uk/lloydsshares/lloyds-share-offer-press-releases/location.

Reply:
Yeah, it's on my list of things to fix next.